builtin/game/chatcommands.lua

```diff
@@ -815,12 +815,18 @@ core.register_chatcommand("days", {
 })
 
 core.register_chatcommand("shutdown", {
+    params = "<delay_in_seconds(0..inf)> | Leave blank for normal shutdown",
```

Comment:
maybe this improvement can be added in another PR later, to have kiss PR instead of huge PRs :) as here it require to add new features to minetest lua core